fbcp-ili9341.cpp

```diff
@@ -47,6 +47,45 @@ int CountNumChangedPixels(uint16_t *framebuffer, uint16_t *prevFramebuffer)
 uint64_t displayContentsLastChanged = 0;
 bool displayOff = false;
 
+#ifdef LOW_BATT_PIN
+bool lowBatt = false;
+uint64_t lowBattLastPolled = 0;
+
+// Battery icon from: https://github.com/martinohanlon/grrl-bat-monitor
+int batt_icon [20][35] = {
```

Comment:
This PR mixes variables with `lowerCamelCase` and `under_score` naming - the convention has been to use `lowerCamelCase` throughout, so perhaps `lowBatteryIcon`?